### PR TITLE
 query multiple tables

### PR DESCRIPTION
**Pull request by Richez003**
created multiply tables
Written the following queries (using JOIN);
1. What animals belong to Melody Pond?
2. List of all animals that are pokemon (their type is Pokemon).
3. List all owners and their animals, remember to include those that don't own any animal.
4. How many animals are there per species?
5. List all Digimon owned by Jennifer Orwell.
6. List all animals owned by Dean Winchester that haven't tried to escape.
7. Who owns the most animals?
 included screenshot 
Screenshot 2022-07-22 163724
Screenshot 2022-07-22 163746
Screenshot 2022-07-22 163813
Screenshot 2022-07-22 163826
Screenshot 2022-07-22 163850
Screenshot 2022-07-22 163908
Screenshot 2022-07-22 163928
Screenshot 2022-07-22 163949
Screenshot 2022-07-22 164001
Screenshot 2022-07-22 164914
Screenshot 2022-07-22 164933
Screenshot 2022-07-22 165258

### CODE REVIEW

File: schema.sql [hunk 1]
```diff
@@ -8,4 +8,26 @@ CREATE TABLE animals (
...
+ALTER TABLE animals DROP COLUMN species;
+ALTER TABLE animals ADD COLUMN species_id;
+ALTER TABLE animals ADD COLUMN INT;
+
+
+/*owners table*/
+CREATE TABLE owners(
+full_name VARCHAR(100) NOT NULL,
+owners_id INT GENERATED ALWAYS AS IDENTITY,
+PRIMARY KEY(owners_id));
```
Comment (DeliceLydia) [inline schema.sql]: - [x] Please add `age` column in this table

File: schema.sql [hunk 2]
```diff
@@ -8,4 +8,26 @@ CREATE TABLE animals (
...
+
+alter table animals drop column owner_id;
```
Comment (DeliceLydia) [inline schema.sql]: - [x] Here you should remove column `species` you can do:
`ALTER TABLE REMOVE species;`

File: data.sql [hunk 3]
```diff
@@ -36,4 +33,30 @@ INSERT INTO animals (name, date_of_birth, neutered, escape_attempts, weight_kg)
...
+/*owners data*/
+INSERT INTO owners (full_name, age) VALUES ('Jennifer Orwell', 19);
+INSERT 0 1
+INSERT INTO owners (full_name, age) VALUES ('Bob', 45);
+INSERT 0 1
+INSERT INTO owners (full_name, age) VALUES ('Melody Pond', 77);
+INSERT 0 1
+INSERT INTO owners (full_name, age) VALUES ('Dean Winchester', 14);
+INSERT 0 1
+INSERT INTO owners (full_name, age) VALUES ('Jodie Whittaker', 38);
```
Comment (DeliceLydia) [inline data.sql:51]: - [x] Please add this data missing:
```sql
 INSERT INTO owners (full_name, age)
 VALUES ('Sam Smith', 34);
```

**Review by DeliceLydia — CHANGES_REQUESTED** [review]
### Hi @Richez003, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._



### Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)

File: schema.sql (hunk 1 above)
Comment (Richez003) [reply schema.sql]: done

File: schema.sql (hunk 2 above)
Comment (Richez003) [reply schema.sql]: done

File: data.sql (hunk 3 above)
Comment (Richez003) [reply data.sql:51]: done

**Review by phelian23 — APPROVED** [review]
Hi @Richez003, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**